class CardNOTFoundError(Exception):
    """Exception raised for errors during data processing."""
    print('Card Not Found')